lib/elements/dom-repeat.js

```diff
@@ -9,19 +9,18 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 */
 import { PolymerElement } from '../../polymer-element.js';
 
-import { TemplateInstanceBase as TemplateInstanceBase$0, templatize, modelForElement as modelForElement$0 } from '../utils/templatize.js';
+import { TemplateInstanceBase, templatize, modelForElement as modelForElement$0 } from '../utils/templatize.js';
```

Comment:
Add `// eslint-disable-line no-unused-vars` at the end

Reply:
Done.